python_files/python_server.py

```diff
@@ -23,8 +26,16 @@ def print_log(msg: str):
     send_message(json.dumps({"jsonrpc": "2.0", "method": "log", "params": msg}))
 
 
-def send_response(response: str, response_id: int):
-    send_message(json.dumps({"jsonrpc": "2.0", "id": response_id, "result": response}))
+def send_response(response: str, response_id: int, execution_status: bool = True):
+    send_message(
+        json.dumps(
+            {
+                "jsonrpc": "2.0",
+                "id": response_id,
+                "result": f"Execution status: {execution_status!s}, Response: {response}",
```

Comment:
```suggestion
                "result": { "status": execution_status, "response": response,
```
You can send status as a boolean right?

Reply:
I don't see why this should be a string

Reply:
Aha, that saves pain from having to regex this on typescript side! Thank you :) 
Did not realize we can send in object {} as a value of "result"